Publish Unit 0, Unit 1 and Bonus Unit 1 (#99)
huggingface/deep-rl-class: units/en/unit1/conclusion.mdx

# Conclusion [[conclusion]]

> Congrats on finishing this chapter! **That was the biggest one**, and there was a lot of information. And congrats on finishing the tutorial. You’ve just trained your first Deep RL agents and shared it on the Hub 🥳.

Reviewer: ```suggestion
Congrats on finishing this unit! **That was the biggest one**, and there was a lot of information. And congrats on finishing the tutorial. You’ve just trained your first Deep RL agents and shared it with the community! 🥳
```

> That’s **normal if you still feel confused with all these elements**. This was the same for me and for all people who studied RL.

Reviewer: ```suggestion
It's **normal if you still feel confused with some of these elements**. This was the same for me and for all people who studied RL.
```

**Take time to really grasp the material** before continuing. It’s important to master these elements and having a solid foundations before entering the fun part.

> Naturally, during the course, we’re going to use and explain these terms again, but it’s better to understand them before diving into the next chapters.

Reviewer: ```suggestion
Naturally, during the course, we’re going to use and explain these terms again, but it’s better to understand them before diving into the next units.
```

> In the next chapter, we’re going to reinforce what we just learn by **training Huggy the Dog to fetch the stick**.

Reviewer: ```suggestion
In the next (bonus) unit, we’re going to reinforce what we just learned by **training Huggy the Dog to fetch the stick**.
```
